queries.sql

```diff
@@ -44,3 +44,13 @@ SELECT species.name, COUNT(animals.name) FROM animals JOIN species ON species.id
 SELECT animals.name FROM animals JOIN species ON species.id = animals.species_id JOIN owners ON owners.id = animals.owner_id WHERE owners.full_name = 'Jenniffer Orwell' AND species.name = 'Digimon';
 SELECT animals.name FROM animals JOIN owners ON owners.id = animals.owner_id WHERE animals.escape_attempts = 0 AND owners.full_name = 'Dean Winchester';
 SELECT COUNT(owners.full_name), owners.full_name FROM animals JOIN owners ON owners.id = animals.owner_id GROUP BY owners.full_name ORDER BY COUNT(owners.full_name) DESC;
+
+SELECT animals.name, visits.visit_date FROM animals JOIN visits ON visits.animal_id = animals.id JOIN vets ON vets.id = visits.vet_id WHERE vets.name = 'William Tacher' ORDER BY visits.visit_date ASC LIMIT 1;
+SELECT animals.name, visits.visit_date FROM animals JOIN visits ON visits.animal_id = animals.id JOIN vets ON vets.id = visits.vet_id WHERE vets.name = 'Stephanie Mendez' GROUP BY (visits.visit_date, animals.name);
+SELECT vets.name, species.name FROM vets LEFT JOIN specialization ON specialization.vet_id = vets.id LEFT JOIN species ON species.id = specialization.species_id;
+SELECT animals.name, visits.visit_date FROM animals JOIN visits ON visits.animal_id = animals.id JOIN vets ON vets.id = visits.vet_id WHERE visits.visit_date BETWEEN '2020-4-1' AND '2020-8-30';
+SELECT animals.name, COUNT(visit_date) FROM animals JOIN visits ON visits.animal_id = animals.id JOIN vets ON vets.id = visits.vet_id GROUP BY animals.name ORDER BY COUNT(visit_date) DESC;
+SELECT animals.name, visits.visit_date FROM animals JOIN visits ON visits.animal_id = animals.id JOIN vets ON vets.id = visits.vet_id WHERE vets.name = 'Maisy Smith' ORDER BY visit_date ASC LIMIT 1;
+SELECT animals.name, animals.date_of_birth, animals.escape_attempts, neutered, weight_kg, vets.name, vets.age, vets.date_of_graduation, visit_date FROM animals JOIN visits ON animals.id = visits.animal_id JOIN vets ON vets.id = visits.vet_id ORDER BY visit_date DESC LIMIT 1;
+SELECT animals.name, animals.species_id as animal_species_id, visit_date, specialization.species_id as vet_species_id, vets.id, vets.name FROM animals JOIN visits ON animals.id = visits.animal_id JOIN vets ON vets.id = visits.vet_id JOIN specialization ON vets.id = specialization.vet_id WHERE animals.species_id != specialization.species_id;
```

Comment:
- [ ] It would be better if we write a query for -> `What specialty should Maisy Smith consider getting? Look for the species she gets the most.`. **HINT**: Use COUNT to get the species she gets the most